### PR TITLE
linuxHelpers.cpp fixes

### PR DESCRIPTION
**Pull request by ckane**
Inconsistent return value as well as fix up the LARGE_INTEGER struct initialization in `GetProcess`

### CODE REVIEW

File: linuxHelpers.cpp [hunk 1]
```diff
@@ -623,7 +623,7 @@ LARGE_INTEGER GetLogonTime(
     CONST LUID* user_luid
     )
 {
-    LARGE_INTEGER result = {0,};
+    LARGE_INTEGER result = {{0},};
```
Comment (kesheldr) [inline linuxHelpers.cpp:626]: Wouldn't
LARGE_INTEGER result = {0};
make more sense? i.e. initialize the QuadPart? The comma is probably the error here.
But appreciate your input; let me know and I'll approve/merge.

**Review by kesheldr — CHANGES_REQUESTED** [review]

File: linuxHelpers.cpp (hunk 1 above)
Comment (ckane) [reply linuxHelpers.cpp:626]: That's a good point, I was just trying to mimic the existing initialization behavior while also following what the -Werror hint was telling me. I'll update with the modification.

**Comment by ckane** [comment]
@kesheldr I just did an ammended commit to use your suggestion (sort of). The value `{{0}}` is accepted by the compiler to resolve `-Wmissing-braces` but simply providing `{0}` is not acceptable. The comma appears to be unnecessary though.

**Review by kesheldr — APPROVED** [review]
Thanks for revisiting this. I guess one set of curly brackets for the struct and another for the union.

**Pull request merged**